packages/filebrowser/src/listing.ts

```diff
@@ -806,6 +809,91 @@ export class DirListing extends Widget {
     }
   }
 
+  // Update the modified column's size
+  updateModifiedSize(node: HTMLElement) {
```

Comment:
Let's minimize API surface so that we can change the signature easily in the future.

```suggestion
  /*
   * Update the modified column's size
   */
  private _updateModifiedSize(node: HTMLElement) {
```

Reply:
`protected` would be fine with me too.